bigtable/google/cloud/bigtable/retry.py

```diff
@@ -0,0 +1,168 @@
+"""Provides function wrappers that implement retrying."""
+import random
+import time
+import six
+
+from google.cloud._helpers import _to_bytes
+from google.cloud.bigtable._generated import (
+    bigtable_pb2 as data_messages_v2_pb2)
+from google.gax import config, errors
+from grpc import RpcError
+
+
+_MILLIS_PER_SECOND = 1000
+
+
+class ReadRowsIterator():
```

Comment:
Please inherit classes from `object`. (We do support Python 2.7 still, so this difference does matter.)